store/modules/settings/actions.js

```diff
@@ -1,30 +1,41 @@
 import { call_axios_get_from_vuex, call_axios_post_from_vuex } from "@/js_utils/axios_helpers.js";
 
 export default {
-  async update_settings(context) {
-    const {
-      active_user_index,
-      user_accounts,
-      auto_upload,
-      auto_delete,
-      pulse3d_version_selection_index,
-      pulse3d_versions,
-    } = this.state.settings;
-    const { customer_id, user_password, user_name } = user_accounts[active_user_index];
-
-    const selected_pulse3d_version = pulse3d_versions[pulse3d_version_selection_index];
+  async update_settings(context, user_details) {
+    const { auto_upload, auto_delete, pulse3d_focus_idx } = user_details;
+    const selected_pulse3d_version = this.state.settings.pulse3d_versions[pulse3d_focus_idx];
 
     const url = "http://localhost:4567/update_settings";
     const params = {
-      customer_id,
-      user_name,
-      user_password,
       auto_upload,
       auto_delete,
       pulse3d_version: selected_pulse3d_version,
     };
 
-    return await call_axios_get_from_vuex(url, context, params);
+    const { status } = await call_axios_get_from_vuex(url, context, params);
+    if (status === 204) {
+      this.commit("settings/set_auto_upload", auto_upload);
+      this.commit("settings/set_auto_delete", auto_delete);
+      this.commit("settings/set_pulse3d_version_selection_index", pulse3d_focus_idx);
+    }
+  },
+  async login_user(context, user_details) {
+    const { customer_id, user, password } = user_details;
+
+    const url = "http://localhost:4567/login";
```

Comment:
Is this for local testing?

Reply:
nope, our FE connects to our BE service using this url.. we use the k8s domains when using the dashboard